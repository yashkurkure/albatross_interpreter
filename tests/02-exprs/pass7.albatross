printint(200+100+50+50+70+3);
printint(200-100-50-50-70-3);
printint(200|100|50|50|70|3);
printint(200&100&50&50&70&3);
printint(200^100^50^50^70^3);
exit(0);
