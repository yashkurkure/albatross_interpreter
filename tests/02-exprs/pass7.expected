473-732550233
